demo/src/test/java/com/example/test/pages/KBBMOOCProject/WhatsMyCarWorthPage/WhatsMyCarWorthPage.java

```diff
@@ -0,0 +1,136 @@
+        
+        return errorMessageString;
+    }
+    public boolean compareErrorMessage(String errorMessage) {
+        if(getErrorMessageString().equals(errorMessage) ){
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Option License Plate
+     * @throws InterruptedException
+     */
+    public void clickOnLicensePlateCheckbox() throws InterruptedException {
+        keyword.scrollAndWaitToClick(chkLicensePlate);
+    }
+    public void sendKeysToTextboxes(String plate, String invalidEmail) throws InterruptedException {
+        WebElement txtPlate = keyword.findWebElementByXpath("//input[@data-lean-auto='plateInput']");
+        WebElement txtEmail = keyword.findWebElementByXpath("//input[@data-lean-auto='emailLpInput']");
+        keyword.sendKeys(txtPlate, plate);
+        keyword.sendKeys(txtEmail, invalidEmail);
+    }
+    public void selectState() throws InterruptedException {
+        WebElement ddlState = keyword.findWebElementByXpath("//select[@placeholder='State']");
+        keyword.scrollAndWaitToClick(ddlState);
+        
+    }
+    public void selectHIoption() throws InterruptedException{
+        WebElement optHI = keyword.findWebElementByXpath("//option[@value='HI']");
+        keyword.scrollAndWaitToClick(optHI);
+    }
+    public void clickOnLicensePlateGoButton() throws InterruptedException{
+        WebElement btnLicensePlateGo = keyword.findWebElementByXpath("//button[@data-testid='lpSubmitBtn']");
+        keyword.scrollAndWaitToClick(btnLicensePlateGo);
+        
+    }
+    public String getEmailErrorMessageString(){
+        WebElement spanEmailErrorMessage = keyword.findWebElementByXpath("//span[@data-automation='error-message']");
+        String emailErrorMessageString = spanEmailErrorMessage.getText();
+        
+        return emailErrorMessageString;
+    }
+    public boolean compareEmailErrorMessage(String emailErrorMessage) {
+        if(getErrorMessageString().equals(emailErrorMessage) ){
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Option Make/Model
+     * @throws InterruptedException
+     */
+    public void clickOnMakeModelCheckbox() throws InterruptedException {
+        keyword.scrollAndWaitToClick(chkMakeModel);
+    }
+    public void selectYear() throws InterruptedException {
+        WebElement ddlYear = keyword.findWebElementByXpath("//select[@placeholder='Year']");
+        keyword.scrollAndWaitToClick(ddlYear);
+      
+        WebElement valueOfYear = keyword.findWebElementByXpath("//option[@value='2016']");
+        keyword.scrollAndWaitToClick(valueOfYear);
+    }
+    public void selectMake() throws InterruptedException {
+        WebElement ddlMake = keyword.findWebElementByXpath("//select[@placeholder='Make']");
+        keyword.scrollAndWaitToClick(ddlMake);
+       
+        WebElement valueOfMake = keyword.findWebElementByXpath("//option[@value='18']");
```

Comment:
should write the web element locator in separate line then call into a method/function